docs/docusaurus/docs/application_integration_support.md

```diff
@@ -2,98 +2,69 @@
 
-| GX Cloud                    | GX OSS                        |
-|-----------------------------|-------------------------------|
-| N/A                         | 0.17<br/>0.18                 | 
+| Integration Type      | GX Cloud²                 | GX Core                                                                       | Community                         |
+| --------------------- | ------------------------- | ----------------------------------------------------------------------------- | --------------------------------- |
+| Data Sources¹         | Snowflake<br/> PostgreSQL | Snowflake<br/>PostgreSQL<br/>Sqlite<br/>Databricks (SQL)<br/>Spark<br/>Pandas | Redshift<br/>MSSQL<br/>MySQL<br/> |
+| Configuration Stores³ | In-app                    | File system                                                                   | None                              |
+| Actions               | Slack                     | Slack ⁴<br/>Email                                                             | None                              |
+| Credential Store      | Environment variables     | Environment variables <br/> YAML⁴                                             | None                              |
+| Orchestrator          | Airflow ⁵                 | Airflow ⁵                                                                     | None                              |
 
-### Integrations
-
-The following table defines the supported GX Cloud and GX OSS integrations.
-
-
-| Integration Type     | GX Cloud    | GX OSS                                                     |
-|----------------------|-------------|------------------------------------------------------------|
-| Data Source          | Snowflake ¹ | Snowflake<br/>Generic SQL ²                                |
-| Configuration Stores | N/A         | Filesystem<br/>AWS S3<br/>GCS Buckets<br/>Azure Blob Store |
-| Notifications        | N/A ³       | Slack ⁴<br/>email<br/>Microsoft Teams<br/>PagerDuty        |
-| Orchestrator         | Airflow ⁵   | Airflow ⁵                                                  |
-
-
-¹ Support for BigQuery will be available in a future release.<br/>
-² Connection strings for various SQL Data Sources are supported, but dialect-specific SQL commands are not.<br/>
-³ Support for Zapier will be available in a future release.<br/>
-⁴ When used with the GX OSS library.<br/>
-⁵ Although only Airflow is supported, GX Cloud and GX OSS should work with any orchestrator that executes Python code.
+¹ We've also seen GX work with the following data sources in the past but we can't guarentee ongoing compatibility. These data sources include Clickhouse, Vertica, Dremio, Teradata, Athena, EMR Spark, AWS Glue, Microsoft Fabric, Trino, Pandas on (S3, GCS, Azure), Databricks (Spark), and Spark on (S3, GCS, Azure).<br/>
+² Support for BigQuery in GX Cloud will be available in a future release.<br/>
```

Comment:
This footnote doesn't appear to be in use in the above table.  You may be missing BigQuery in the above table, or need to remove this footnote and renumber the others.

Reply:
updated